### PR TITLE
Update Aquasec's Helm Charts

### PR DESCRIPTION
**Pull request by leonidlm**

### CODE REVIEW

File: aquasec-enforcer/templates/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-cluster-role
+  labels:
+    app: {{ .Release.Name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+# TODO: Make role's permissions much more specific for enforcer needs
```
Comment (leonidlm) [inline aquasec-enforcer/templates/rbac.yaml]: @oranmoshai please specify better scoped permissions

File: aquasec-enforcer/values.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+rbac:
+  enabled: true
+  roleRef:
+
+image:
+  repository: aquasec/agent
+  tag: 3.2
+  pullPolicy: IfNotPresent
+
+# TODO: Fill-in the correct liveness and readiness probes
```
Comment (leonidlm) [inline aquasec-enforcer/values.yaml]: @oranmoshai please provide the needed probes

File: aquasec-server/templates/gate-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      name: {{ .Release.Name }}-gate
+    spec:
+      serviceAccount: {{ .Release.Name }}-account
+      containers:
+      - name: gate
+        image: "{{ .Values.imageCredentials.repositoryUriPrefix }}/{{ .Values.gate.image.repository }}:{{ .Values.gate.image.tag }}"
+        imagePullPolicy: "{{ .Values.gate.image.pullPolicy }}"
+        env:
+        # TODO: What default value should we use for SCALOCK_GATEWAY_PUBLIC_IP
+        - name: SCALOCK_GATEWAY_PUBLIC_IP
```
Comment (leonidlm) [inline aquasec-server/templates/gate-deployment.yaml]: @oranmoshai can you please verify if we must specify this env var, and if not, what are the implications.

File: aquasec-server/templates/rbac.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-cluster-role
+  labels:
+    app: {{ .Release.Name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+# TODO: Make role's permissions much more specific for server component needs
```
Comment (leonidlm) [inline aquasec-server/templates/rbac.yaml]: @oranmoshai please help with rbac roles here too

File: aquasec-server/values.yaml [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  readinessProbe:
+    exec:
+      command:
+      - sh
+      - -c
+      - exec pg_isready --host $POD_IP
+    initialDelaySeconds: 5
+    timeoutSeconds: 3
+    periodSeconds: 5
+  # TODO: Specify the minimal DB resource requirements
```
Comment (leonidlm) [inline aquasec-server/values.yaml]: @leonidlm set this to 1cpu and 1g

File: aquasec-server/values.yaml [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+gate:
+  image:
+    repository: aquasec/gateway
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+    externalPort: 3622
+  replicaCount: 1
+  # TODO: Fill-in the correct liveness and readiness probes
```
Comment (leonidlm) [inline aquasec-server/values.yaml]: @oranmoshai can you please spcify the probes for the gate component

File: aquasec-server/values.yaml [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+  enabled: false
+  image:
+    repository: aquasec/scanner-cli
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  # TODO: Are there any defaults for the following values?
+  user:
+  password:
+  replicaCount: 1
+  # TODO: Fill-in the correct liveness and readiness probes and basic resource configuration
```
Comment (leonidlm) [inline aquasec-server/values.yaml]: @oranmoshai can you please provide the probes for scanner too

File: aquasec-enforcer/templates/rbac.yaml (hunk 1 above)
Comment (oranmoshai) [reply aquasec-enforcer/templates/rbac.yaml]: Enforcer do not need cluster role he is not working with kubernetes API or using other kubernetes objects.

File: aquasec-enforcer/values.yaml (hunk 2 above)
Comment (oranmoshai) [reply aquasec-enforcer/values.yaml]: Currently we do not have healthcheck endpoint in the enforcer we will add it to the roadmap

File: aquasec-server/templates/gate-deployment.yaml (hunk 3 above)
Comment (oranmoshai) [reply aquasec-server/templates/gate-deployment.yaml]: Yes it is must for kubernetes you need to add the service name of the gateway deployment.

File: aquasec-server/templates/rbac.yaml (hunk 4 above)
Comment (oranmoshai) [reply aquasec-server/templates/rbac.yaml]: Currently there is no need for rbac in this section.

File: aquasec-server/values.yaml (hunk 6 above)
Comment (oranmoshai) [reply aquasec-server/values.yaml]: Currently there is no probe for gateway we will add it to the roadmap

File: aquasec-server/values.yaml (hunk 7 above)
Comment (oranmoshai) [reply aquasec-server/values.yaml]: Currently there is no probe for scanner we will add it to the roadmap

**Pull request merged**